support/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -105,93 +119,169 @@ Note: If you would like this tracking domain to be the default, please add `"def
 
-    * Under Distribution Settings, fill in **Alternate Domain Names (CNAMEs)** with your custom tracking domains (i.e., www.customtrackingdomain.com).	
-        * **Note:** Can add up to 100 domains.	
+        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_create_distribution.png)
 
-        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_cnames.png)	
 
-    * Under Default Cache Behavior Settings, set **Query String Forwarding and Caching** to "Forward all, cache based on all".	
+1. <a name="cname"></a> Create, or update, a CNAME record with your DNS service to route queries for tracking domain(s) with your CloudFront distribution ID. This will be specific to your DNS service.
 
-        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_query_string_forward.png)	
+   * Get the "Domain Name" for your distribution.
 
-    * Create, or update, a CNAME record with your DNS service to route queries for tracking domain(s) with your CloudFront distribution ID.	
+        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_dist_domain.png)
 
-    * Under Distribution Settings, for "SSL Certificate", select **Custom SSL Certificate** - Upload certificates as needed.	
+    * Create the CNAME record within your DNS service (this will be specific to your provider). If you have a TTL (time to live) field, we suggest to set this to 1 hour.
 
-        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_custom_ssl_cert.png)	
+        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_domain_provider_xyz_cname.png)
 
-    * At the bottom of the page, press **Create Distribution**.	
+        _Example DNS provider CNAME setup_
 
-        ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_create_distribution.png)	
+        You can verify that the routing is successful using `ping` on your created record. You should see a response from `xyzzy.cloudfront.net`.
 
-5. On the **Create Distribution** page, under Default Cache Behavior Settings, it is recommended to set a custom whitelist for the `User-Agent` header.  Set the configuration **Cache Based on Selected Request Headers** to "Whitelist".  At this point you will need to "Add Custom" for `User-Agent`.  This allows `User-Agent` data to be present in your engagement events received from SparkPost.	
+### <a name="sparkpost-app"></a> In the SparkPost app
 
-    ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_whitelist_cache.png)	
+1. Log into SparkPost app, or use the SparkPost [Tracking Domain API](https://developers.sparkpost.com/api/tracking-domains/#tracking-domains).
 
-6. Log into SparkPost app use the SparkPost [Tracking Domain API](https://developers.sparkpost.com/api/tracking-domains/#tracking-domains).	
+1. Add **custom tracking domain**. It can't be verified just yet.
```

Comment:
Should these be universal steps at the end of the document?  Also, once the CDN and DNS is updated, they can verify the tracking domain.

Reply:
Yes. We had unnecessary repetition in each section. Done.